Button/tests/Button-specs.js

```diff
@@ -30,6 +30,24 @@ describe('Button Specs', () => {
 		expect(button).toHaveClass(expected);
 	});
 
+	test('should have \'joinedCenter\' class', () => {
```

Comment:
Is there a reason you wrote `\'` instead of  backtick? 
For readability, ` looks better.
If there is no reason, please fix this throughout this PR
```suggestion
	test('should have `joinedCenter` class', () => {
```